app/assets/javascripts/backbone/views/timeline_batch_actions_view.coffee

```diff
@@ -0,0 +1,27 @@
+class MS.Views.TimelineBatchActionsView extends Backbone.View
+
+  el: 'body'
+
+  events: (params) ->
+    "confirm:complete .batch-actions .batch-action": 'adjust_form'
+    "click #adjust-dates-confirm": 'hide_adjust_dates_modal'
+    "shown.bs.modal #adjust-dates": 'show_adjust_dates_modal'
```

Comment:
I believe our styleguide says to prefer single quotes in JS. Can you please keep that in mind going forward but ok to not fix these for now.